swift/ql/src/queries/Security/CWE-311/CleartextStorageDatabase.ql

```diff
@@ -20,7 +20,7 @@ import DataFlow::PathGraph
 /**
  * An `Expr` that is stored in a local database.
```

Comment:
This comment is now slightly wrong. `Stored` is no longer an `Expr`, but a `DataFlow::Node`.

Reply:
Good point.  Updated.